source/test/diff.ts

```diff
@@ -0,0 +1,74 @@
+import {verifyWithDiff} from './fixtures/utils';
+import execa, {ExecaError} from 'execa';
+import path from 'path';
+import test from 'ava';
+import tsd from '..';
+
+test('diff', async t => {
+	const diagnostics = await tsd({cwd: path.join(__dirname, 'fixtures/diff')});
+
+	verifyWithDiff(t, diagnostics, [
+		[8, 0, 'error', 'Parameter type `{ life?: number | undefined; }` is declared too wide for argument type `{ life: number; }`.', {
+			expected: '{ life?: number | undefined; }',
+			received: '{ life: number; }',
+		}],
+		[9, 0, 'error', 'Parameter type `FooFunction` is not identical to argument type `() => number`.', {
+			expected: '(x?: string | undefined) => number',
+			received: '() => number',
+		}],
+		[10, 0, 'error', 'Parameter type `FooType` is declared too wide for argument type `Required<FooType>`.', {
+			expected: '{ foo?: "foo" | undefined; }',
+			received: '{ foo: "foo"; }',
+		}],
+		[11, 0, 'error', 'Parameter type `Partial<FooInterface>` is declared too wide for argument type `Required<FooInterface>`.', {
+			expected: '{ [x: string]: unknown; readonly protected?: boolean | undefined; fooType?: FooType | undefined; id?: "foo-interface" | undefined; }',
+			received: '{ [x: string]: unknown; readonly protected: boolean; fooType: FooType; id: "foo-interface"; }',
+		}],
+		[13, 0, 'error', 'Argument of type `{ life: number; }` is assignable to parameter of type `{ life?: number | undefined; }`.', {
+			expected: '{ life?: number | undefined; }',
+			received: '{ life: number; }',
+		}],
+		[18, 0, 'error', 'Documentation comment `This is a comment.` for expression `commented` does not include expected `This is not the same comment!`.', {
+			expected: 'This is not the same comment!',
+			received: 'This is a comment.',
+		}]
+	]);
+});
+
+test('diff cli', async t => {
+	const file = path.join(__dirname, 'fixtures/diff');
+
+	const {exitCode, stderr} = await t.throwsAsync<ExecaError>(execa('dist/cli.js', [file, '--show-diff']));
+	const expectedLines = stderr.trim().split('\n').slice(1).map(line => line.trim());
```

Comment:
Which is what this line is supposed to remove, right?

Maybe add this snippet to see what the CI reports is the first line:

```ts
console.log(stderr.trim().split('\n').at(0));
```

(Or log the whole array if that's easier for debugging)

Reply:
Thanks for the hint ;) Fixed, this is because of the addition of the error message and stack trace recently added in https://github.com/SamVerschueren/tsd/blob/621aad986c636ff511293706455720b006598945/source/cli.ts#L61 which adds some lines to the output.